src/test/java/com/cegeka/academy/repository/ChallengeCategoryTest.java

```diff
@@ -0,0 +1,48 @@
+package com.cegeka.academy.repository;
+
+import com.cegeka.academy.AcademyProjectApp;
+import com.cegeka.academy.domain.ChallengeCategory;
+import com.cegeka.academy.domain.Interest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(classes = AcademyProjectApp.class)
+@Transactional
+class ChallengeCategoryTest {
+
+    @Autowired
+    private ChallengeCategoryRepository challengeCategoryRepository;
+
+    @Test
+    public void testAddChallengeCategory(){
+        ChallengeCategory challengeCategory =  new ChallengeCategory();
+        challengeCategory.setName("category1");
+        challengeCategory.setDescription("description1");
+        challengeCategoryRepository.save(challengeCategory);
+        assertThat(challengeCategoryRepository.findAll().size()).isEqualTo(1);
```

Comment:
Instanta de tip ChallengeCategory se repeta in toate testele. Cel mai bine instantiezi acest obiect intr-un @Before astfel incat sa fie creat pentru fiecare test rulat.